decidim-core/spec/lib/dependency_resolver_spec.rb

```diff
@@ -124,7 +125,6 @@ module Decidim
       decidim-sortitions
       decidim-surveys
       decidim-system
-      decidim-templates
```

Comment:
I understand that you're removing decidim-templates because it was two times. Great finding!

Can you sort all of this alphabetically so this doesn't happen again? I'm talking about the whole array, from line 108 tot 132. Thanks